lib/plugin.js

```diff
@@ -13,7 +13,7 @@ var REQUIRED_OPTS = {
 var DEFAULT_PROTOCOL = 'http';
 
 module.exports = function (gemini, opts) {
-    if (opts.enabled === false) {
+    if (!_.isObject(opts) || opts.enabled === false) {
```

Comment:
Because it is the only way to disable plugin using env variable.  We can't just pass `gemini_system_plugin_tunnel_enabled=false` because config-parser handles nothing but `gemini_system_plugin_tunnel`, and it parses it as a string.
@SwinX it would be nice to have it in README